meetings/2021-07-28.md

```diff
@@ -49,30 +49,47 @@ Sarven: Because you're authorized (or not).
-Currently, with WAC, it can't find out.
-Because right now, you follow a link, and the only person that follows the link and wants to read the ACL is the controller. There is no other way of saying it.
+Currently, with WAC, it can not find out.
+Because right now, if you follow a link from somewhere on the web to a Pod, then the only agent that can follow the ACL link is the Controller.
 If I am a friend of yours and follow links and want to read something, I have no idea what identity to use in order to access resources.
-This is not a decentralised network, it is 1990s PC.
-There are a ton of experiments that put a user at the center of using a cloud-based system.
-That is not decentralisation.
-We can't make the same mistake.
-The question is, how do I, coming across a link, learn what the access control rules are?
+So if the only server able to read my resources is myself
+then we do not have a decentralized network: we have a 1990s PC.
+
+There have been many experiments in giving the user back control of their data that make this mistake: they put a user at the center, but then fail to make a useable protocol that makes exchanging data easy. For example, the original [mes infos](http://mesinfos.fing.org) project I mentioned recently, for personal cloud computing in france had this type of architecture. The user could save his own data, but the protocol for exchanging this with anyone else was some JSON RPC style.
+
+1980ies PCs is not the decentralization we are aiming at.
+We do not want to make the same mistake. We want to be able to share data, so that we can build a network effect. 
+(True: the current WAC does allow the sharing of data, but only if it is public, so we are somewhat beyond 1980s).
+The question is, how do I, coming across a link to a protected resource, learn what the access control rules are?
 There is a simple answer, which would be making the ACLs more readable than the resources.
 
 Sarven: We can introduce a new access mode like ControlRead. Or using problem details, e.g., on 403, payload indicating required credentials (over 18), we can describe the structure/semantics as well on the Protocol level.
 
-Henry: We need the access control rules to be readable by the client.
+Henry: Yes, ControlRead or ACLs on ACLs would do.
+We need the access control rules to be readable by the client.
 
 Elf: I don't think that's correct. I strongly disagree with the statement you make that it is holding in all cases.
 
-Henry: OK, but even me with my client that is open source, I should be able to edit my own content that I want to see.
+Henry: OK, not all ACLs MUST be readable publically. But it should be possible to set the visibility of ACLs. 
+
+Elf: All browsers I know of have a notion of profile that limits the identity I use to log into a site to one identity. None of them have a feature that allows you to switch between identities whilst using a profile. 
```

Comment:
Perhaps worth noting the work OpenLink did with [YouID browser extension](http://youid.openlinksw.com/), which (among other things) allows a TLS session to be bound to one ID (meant to be associated with the Browser, as opposed to the User) while the application login carried across that TLS session may be switched from one UserID to another, working around the browser vendors' failure to provide a means of terminating TLS session without terminating browser process/session.  See --
* [Web, Logic, Sentences, and the Magic of Being You!](https://medium.com/virtuoso-blog/web-logic-sentences-and-the-magic-of-being-you-e2a719d01f73)
* [A Simple Usage Exercise with the YouID Browser Extension](https://medium.com/openlink-software-blog/simple-youid-browser-extension-usage-exercise-57fa3ff6c6b7)